qt/widgets/common/inc/MantidQtWidgets/Common/MantidTreeWidget.h

```diff
@@ -47,7 +47,7 @@ class EXPORT_OPT_MANTIDQT_COMMON MantidTreeWidget : public QTreeWidget {
 
 private:
   QPoint m_dragStartPosition;
-  QWorkspaceDockView *m_dockWidget;
+  QWorkspaceWidget *m_dockWidget;
```

Comment:
It looks as if this is not used anywhere and can be removed, which is good as this dependency shouldn't need to exist.